src/primo/browzine-primo-adapter.js

```diff
@@ -529,6 +557,14 @@ browzine.primo = (function() {
       articlePDFDownloadLinkText = browzine.articleRetractionWatchText || "Retracted Article";
     }
 
+    if (showEocNoticeUI(articleEocNoticeUrl) && !showRetractionWatchUI(articleRetractionUrl)) {
+      directToPDFUrl = articleEocNoticeUrl;
+      pdfIcon = "https://assets.thirdiron.com/images/integrations/browzine-retraction-watch-icon.svg";
+      pdfIconWidth = "15";
+      pdfIconMarginRight = "1.5px";
+      articlePDFDownloadLinkText = browzine.articleExpressionOfConcernText || "Expression of Concern";
```

Comment:
Same with this for a config option - you'll want to tell someone in ops about potentially adding this to publicly-facing config documentation, too.

Although this GitHub repo is publicly accessible, since most people don't need to look at the code, most of the documentation integrators use are the Confluence ones.